flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
:-(

Reply:
https://flathub.org/builds/#/builders/5/builds/7379

Reply:
yes I saw. and that still make me :-(

Reply:
Maybe try building without the git submodules, and build the dependencies separately. The AUR package has [this patch](https://aur.archlinux.org/cgit/aur.git/tree/0001-makepkg-Remove-external-stuff.patch?h=imhex).

Reply:
It seems building FMT separately solved this.